Sources/Cells/BasicTableViewCell/BasicTableViewCell.swift

```diff
@@ -65,22 +65,22 @@ open class BasicTableViewCell: UITableViewCell {
 
     // MARK: - Public methods
 
-    open func configure(with viewModel: BasicTableViewCellViewModel, indexPath: IndexPath? = nil) {
-        titleLabel.text = viewModel.title
+    open func configure(with viewModel: BasicTableViewCellViewModel?, indexPath: IndexPath? = nil) {
```

Comment:
Woops, I just forgot to remove these changes. Although it is nice to have this as an optional because in `prepareForReuse()` you can simply say `configure(with: nil)`. But not in this PR

Reply:
And it would force us to consider displaying an empty cell, which is a good thing